pybay-2025/videos/ai-ip-and-your-code-what-developers-need-to-know.json

```diff
@@ -0,0 +1,29 @@
+{
+  "copyright_text": "",
+  "description": "In a world where AI-generated code is becoming more prevalent, who owns the output? Can AI companies freely use your open-source code as training data? What are the legal ramifications when an AI system infringes upon existing intellectual property rights?Alla will address these critical questions focused on code and IP law. The talk aims to demystify the complex legal landscape developers may find themselves navigating when using AI-generated code.\n\n\nFor more information, see https://pybay.org/\n\nFollow us on LinkedIn https://www.linkedin.com/company/pybay",
```

Comment:
It looks like the YouTube descriptions include boilerplate at the end of each talk description to include links to pybay.org and the LinkedIn page. Otherwise, these seem identical (at first glance) to the text on the conference schedule webpage, which we try to match when possible. If there's still work to be done on this PR (I notice that it's marked as a draft, would you mind removing this bit? If it's otherwise ready to go, I can remove it before merging.

Reply:
It's not a hard and fast rule. It's more important e.g. with very different description text between YouTube and the official schedule, or when speaker and tag info are present within the description but not in the PyVideo metadata fields. Happy to merge this as-is (and / or re-review) once it's out of draft mode.